logstash-core/src/main/java/org/logstash/instrument/witness/PluginsWitness.java

```diff
@@ -20,10 +30,12 @@ public PluginsWitness() {
         this.inputs = new ConcurrentHashMap<>();
         this.outputs = new ConcurrentHashMap<>();
         this.filters = new ConcurrentHashMap<>();
-     }
+        this.codecs = new ConcurrentHashMap<>();
```

Comment:
@jakelandis I just saw we have a bunch of `ConcurrentHashMap`s here, maybe we should initialize them a little smaller (concurrency level 1 should be fine here right? We rarely update the actual map here?)